web/pageComponents/shared/Hero/FiftyFiftyHero.tsx

```diff
@@ -55,7 +55,6 @@ const StyledIngress = styled.div`
 `
 const StyledHeroTitle = styled(TitleText)`
   max-width: 1186px; /* 1920 - (2 * 367) */
-  font-weight: var(--fontWeight-medium);
```

Comment:
why remove this? this will change how the default 50/50 header looks. It should only be removed for the bigTitle variation